### PR TITLE
Add `paymentUrl` support

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

File: src/SwedbankPay/Core/Library/Methods/Card.php [hunk 1]
```diff
@@ -74,6 +75,12 @@ public function initiateCreditCardPayment($orderId, $generateToken, $paymentToke
             ]
         ];
 
+        if ($this->configuration->getData(ConfigurationInterface::CHECKOUT_METHOD) ===
+            ConfigurationInterface::METHOD_SEAMLESS
+        ) {
+            $params['payment']['urls']['paymentUrl'] = $urls->getPaymentUrl();
```
Comment (asbjornu) [inline src/SwedbankPay/Core/Library/Methods/Card.php:81]: What does `$urls->getPaymentUrl()` return? Where is the method defined?

**Comment by asbjornu** [comment]
The tests are failing?

File: src/SwedbankPay/Core/Library/Methods/Card.php (hunk 1 above)
Comment (aait) [reply src/SwedbankPay/Core/Library/Methods/Card.php:81]: @asbjornu It returns the checkout page where js code should be placed. It defined there https://github.com/SwedbankPay/swedbank-pay-woocommerce-core/pull/46/commits/ef0957debba787b66ec2d224e4fad5d0cfdfa143#diff-e4e3b2fe15e4bfb33e862bc52da037baaae2287f76484e46d97f422d0bb83bf4R13
and there
https://github.com/SwedbankPay/swedbank-pay-woocommerce-core/pull/46/commits/ef0957debba787b66ec2d224e4fad5d0cfdfa143#diff-9c236e1e67d9a25d7c8e78ccbe515ead23ecfa910e08d4910db7f16739a79ca7R15

**Comment by aait** [comment]
> The tests are failing?

There's MobilePay doesn't work.
```
{"sessionId":"7550aa7a-fdfb-4480-ab96-b2405469a647","type":"https://api.payex.com/psp/errordetail/forbidden","title":"Operation not allowed","status":403,"instance":"http://api.externalintegration.payex.com/psp/mobilepay/payments","detail":"Not allowed, description in problems node!","problems":[{"name":"UseMobilePayV2","description":"Value UseMobilePayV2 must be True on MobilePay Contract"}]}
```

**Comment by aait** [comment]
> The tests are failing?

It's working now

**Review by asbjornu — APPROVED** [review]

**Pull request merged**